projects/large-messages/upload_img.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+SCW_BUCKET="BUCKET_NAME_FROM_TERRAFORM"
+SCW_NATS_URL="NATS_URL_FORM_TERRAFORM"
```

Comment:
If you set these as Terraform `output`, you can use `SCW_NATS_URL=$(terraform output nats_url)` for example, and the user will not have to modify the script.

Reply:
Oh great ! i didn't know that